roles/autotested/templates/subtests.ini.j2

```diff
@@ -15,7 +15,7 @@ subsubtests = good,wrong_tag,wrong_registry_addr
 config_version = 0.8.5
 generate_generic = yes
 ping_url = "www.redhat.com"
-cmd = 'ping -c 3 -q %(ping_url)s'
+cmd = 'yum install -y iputils && ping -c 3 -q %(ping_url)s'
```

Comment:
This never works for me: my docker containers can't run yum. Does it work for you?